.dev/performance/results.js

```diff
@@ -74,13 +74,43 @@ async function getPerformanceTestResults( branch ) {
 			},
 			invertedResult
 		);
+
 		// eslint-disable-next-line no-console
 		console.table( invertedResult );
 
-		const resultsFilename = testSuite + '-performance-results.json';
+		let resultsTable = '';
+		let headers = '';
+		let iteration = 1;
+		let dataKey;
+		const dataKeys = Object.keys( results[ testSuite ].master );
+
+		for ( const [ key ] of Object.entries( results[ testSuite ] ) ) {
+			if ( iteration === 1 ) {
+				headers += 'index';
+			}
+
+			headers += ' | ' + key;
+
+			if ( iteration === 3 ) {
+				headers += '\\r\\n';
+				headers += ':--- | :---: | :---: | :---:';
+			}
+
+			iteration++;
+		}
+
+		// eslint-disable-next-line
+		for ( dataKey in dataKeys ) {
+			resultsTable += dataKeys[ dataKey ];
+			// eslint-disable-next-line
+			resultsTable += ' | ' + results[ testSuite ][ 'master' ][ dataKeys[ dataKey ] ] + ' | ' + results[ testSuite ][ branch ][ dataKeys[ dataKey ] ] + ' | ' + results[ testSuite ][ 'change %' ][ dataKeys[ dataKey ] ] + '%\\r\\n';
```

Comment:
Do you think this would be more readable using template literals?
```
` | ${ results[ testSuite ][ 'master' ][ dataKeys[ dataKey ] ] } | ${ results[ testSuite ][ branch ][ dataKeys[ dataKey ] ] } | ${ results[ testSuite ][ 'change %' ][ dataKeys[ dataKey ] ] }%\\r\\n`
```

Reply:
@AnthonyLedesma Yes, it would probably would be.